src/style.css

```diff
@@ -0,0 +1,69 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.input {
+  display: flex;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  width: 100%;
+  font-family: inherit;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  border-radius: 3px;
+  outline: none;
+  font-weight: bold;
+}
+
+.input::placeholder {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
+
+.input:focus::placeholder {
+  color: transparent;
```

Comment:
why do you hide placeholder? it fails a test

Reply:
this test didn`t pass before I wrote this part of code